index.js

```diff
@@ -151,7 +151,7 @@ async function autoResetSession(locationid) {
 // This function seeds the state table with a RESET state in case there was a prior unresolved state discrepancy
 if (!helpers.isTestEnvironment()) {
   setInterval(async () => {
-    const locations = await db.getLocations()
+    const locations = await db.getActiveLocations()
```

Comment:
Doesn't check inactive sessions for the auto reset conditions because we don't want to send out any auto-reset messages.